core/commands/cherry_pick.py

```diff
@@ -1,13 +1,40 @@
 import sublime
+from sublime_plugin import TextCommand
 
 from .log import GsLogCommand
 from ..git_mixins.branches import BranchesMixin
+from ...common import util
+
+
+class GsCherryPickHashCommand(TextCommand, BranchesMixin):
```

Comment:
If this command relies on how text is laid out in the branch history view, this class should be placed in that file.